utils/controller_debug.py

```diff
@@ -699,12 +704,13 @@ def TraceSaveDat(dat, fname, separator="  "):
     tv = TraceActiveVars()
 
     fp = open(fname, "w")
-    line = ["time"]
+    line = ["time(sec)"]
     for v in tv:
         line.append(v.name)
     fp.write(separator.join(line) + "\n")
 
     for i in range(len(dat[0])):
+        # First colume is time in seconds
```

Comment:
colume

Reply:
Doh, that's why I keep misspelling volume as volumn.  English is a weird language